src/extensions/starter_dashlet/BoxWithHandle.tsx

```diff
@@ -0,0 +1,32 @@
+import React, { CSSProperties, FC, memo } from 'react';
+import { useDrag } from 'react-dnd';
+import Icon from '../../controls/Icon';
+import { StarterInfo } from '../../util/api';
```

Comment:
please don't import "api" files from the core application. There are at least two occurrences of this in this diff.